src/backend/src/prisma/schema.prisma

```diff
@@ -290,3 +306,21 @@ model Risk {
   deletedByUserId  Int?
   deletedBy        User?     @relation(name: "deletedBy", fields: [deletedByUserId], references: [userId])
 }
+
+model Task {
+  taskId          String        @id @default(uuid())
+  title           String
+  notes           String
+  deadline        DateTime
+  assignees       User[]        @relation(name: "assignedTo")
+  priority        Task_Priority
```

Comment:
Might want a default on this

Reply:
I dont think so because when we create the task it can be created in any of the tabs so well just pass in that value when creating the task